doc/api/sqlite.md

```diff
@@ -538,11 +542,15 @@ exception.
 
 <!-- YAML
 added: v23.8.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/56991
+    description: The `destination` argument now supports Buffer and URL objects.
 -->
 
 * `sourceDb` {DatabaseSync} The database to backup. The source database must be open.
-* `destination` {string} The path where the backup will be created. If the file already exists, the contents will be
-  overwritten.
+* `destination` {string | Buffer | URL} The path where the backup will be created. If the file already exists,
```

Comment:
This should be renamed to `path` because a `URL` called `destination` is confusing.

You should mention that [URI Filenames](https://www.sqlite.org/c3ref/open.html) are supported.

Reply:
Could you share your inputs on this [thread](https://github.com/nodejs/node/pull/56991#discussion_r1965798512)?